### PR TITLE
feat!: Improve the title bar

### LINKED ISSUE #2518: Improve the title bar of main JabRef website
**Is your suggestion for improvement related to a problem? Please describe.**

Sometimes, when I want to open JabRef's GitHub repo, the address bar in my browser proposes the main [JabRef website](https://www.jabref.org/).

Current title bar:
image

Some open-source projects title bars:
image
image
image

Many open source projects put a link to their GitHub repo in the title bar. If we had this button, I could easily navigate to GitHub repo from the main website.

**Describe the solution you'd like**

Add GitHub link to title bar. Additionally, you can add more links.

However, main JabRef website doesn't have a lot of content, so probably only GitHub link would be useful.

**Additional context**

It's a very low priority issue that involves only web coding.

### PR DESCRIPTION
**Pull request by plvzfq-rit**
### 🔗 Linked issue

Resolves #2518 

### 📚 Description

- Reworked the title bar such that the links above direct to sites that might be of interest (Docs, Forums, GitHub, etc.).
- ~Removed some code that did not seem necessary for now like user profiles and the search bar (might add it back though should it become necessary)~ Readded these functionalities back
- Some dropdown menus to the nav bar that shows some other links that users might find interesting (to Docs and Forums)
- Add collapsible functionality to the title bar so that it is mobile responsive

Not really quite necessary, just a nice to have that could aid people going to the site in navigating to other places.

### Previous title bar
image

### Proposed title bar
image

#### New dropdowns

image

image

### CODE REVIEW

**Comment by tobiasdiez** [comment]
Thanks!

> Removed some code that did not seem necessary for now like user profiles and the search bar (might add it back though should it become necessary)

These are not shown on the main start page, but once you login they are shown (try the subpath `/dashboard` - but this section is a bit broken currently). So please don't remove these parts / readd them.

File: components/NavBar.vue [hunk 1]
```diff
@@ -1,249 +1,66 @@
...
+                    class="text-gray-400 hover:text-primary-600 text-lg font-semibold"
+                    to="https://docs.jabref.org/"
+                    >Docs</t-nuxtlink
+                    >
+                    <n-divider vertical />
+                    <t-nuxtlink
+                    active-class="text-gray-400 hover:text-primary-600 text-lg font-semibold"
+                    class="text-gray-400 hover:text-primary-600 text-lg font-semibold"
+                    to="https://builds.jabref.org/"
+                    >Builds</t-nuxtlink
```
Comment (tobiasdiez) [inline components/NavBar.vue:23]: I don't think the nightly builds are that relevant.

Comment (plvzfq-rit) [reply components/NavBar.vue:23]: Ah, I'll remove them then, much thanks+

**Comment by plvzfq-rit** [comment]
I'll readd the functionalities back as soon as I could; apologies for that;;;

**Comment by plvzfq-rit** [comment]
Readded the functionalities and changed out the nightly build with Gitter to partially help with issue #2562. Also made some of the menus dropdowns so that one could go farther.

**Comment by tobiasdiez** [comment]
Thanks a lot for your work @plvzfq-rit. I'm a bit undecided if I like the changes though. The menu was designed to be simple and give access to the most important links for _new_ users. With these changes, there are now a lot of information that new users usually don't care about (they want to have an answer to 2 questions: is it worthwhile to install JabRef, and if so where can I download it).

@JabRef/developers what do you think?

**Comment by koppor** [comment]
> Thanks a lot for your work @plvzfq-rit. I'm a bit undecided if I like the changes though. The menu was designed to be simple and give access to the most important links for _new_ users. With these changes, there are now a lot of information that new users usually don't care about (they want to have an answer to 2 questions: is it worthwhile to install JabRef, and if so where can I download it).
> 
> @JabRef/developers what do you think?

I don't see any before/after screenshots nor any user stories. Hard to tell what I like more.

**Comment by plvzfq-rit** [comment]
Hi @tobiasdiez, @koppor, I've added in a photo of the previous title bar and a photo of the new one; I've also included images of the new dropdowns as well. Apologies for not adding this in sooner

**Comment by koppor** [comment]
I would design it differently, too. The issue also asks for a GitHub-Button, too. Not a "portal".

Please focus on the new "GitHub" button - and fix https://github.com/JabRef/JabRefOnline/issues/2176. That issue is the most urgent one!

**Comment by plvzfq-rit** [comment]
Alright, I'll do just that; I'll close this pull request and focus on the other one which is more apt (#2560 ). I'll also have a look into #2176 . Thank you for y'all's generous input++

**Pull request closed without merging**